examples/errors/mainwindow_funcs.cpp

```diff
@@ -39,11 +39,13 @@ MainWindow::MainWindow() : QMainWindow()
     dataEntity_->setData(xValues_, yValues_, uniformErrors_);
     dataEntity_->setSymbolStyle(Mildred::StyleFactory1D::SymbolStyle::Triangle);
     dataEntity_->setSymbolStyle(Mildred::StyleFactory1D::SymbolStyle::Square);
+    dataEntity_->setSymbolStyle(Mildred::StyleFactory1D::SymbolStyle::Diamond);
 
     ui_.StyleCombo->addItem(QString("Stick"));
     ui_.StyleCombo->addItem(QString("T-Bar Stick"));
     ui_.SymbolCombo->addItem(QString("Triangle"));
     ui_.SymbolCombo->addItem(QString("Square"));
+    ui_.SymbolCombo->addItem(QString("Diamond"));
     ui_.SymbolCombo->addItem(QString("None"));
```

Comment:
I'd suggest reordering these so that the "None" is first, and the default choice:
```suggestion
    ui_.SymbolCombo->addItem(QString("None"));
    ui_.SymbolCombo->addItem(QString("Triangle"));
    ui_.SymbolCombo->addItem(QString("Square"));
    ui_.SymbolCombo->addItem(QString("Diamond"));
```